### DISCUSSION TITLE
links dropdown

### DISCUSSION THREAD

**Discussion by strowi** [grafana/grafana#38239] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Allow links to appear as dropdown-menu, similar to dashboards in the upper right corner.

**Why is this needed**:
Having multiple Links consumes too much space.

regards,
strowi

**Comment by torkelo** [comment]
which links are you talking about?

**Comment by strowi** [comment]
Sorry for not being clearer, i was talking about the dashboard links in the upper right corner - here is a screenshot:

![grafik](https://user-images.githubusercontent.com/589662/111060831-1496e800-84a0-11eb-9221-78dde85b8bd2.png)

Type dashboard can appear as dropdown, but type link not.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
